simtools/layout/array_layout.py

```diff
@@ -524,6 +546,63 @@ def export_telescope_list_table(self, crs_name):
 
         return table
 
+    def export_telescope_list_as_json(self, crs_name):
+        """
+        Return a single-telescope list as simtools-DB-style json.
```

Comment:
I am not sure what "single-telescope list" means. A list with a single telescope in it? or a single list with all the telescopes?